Provide arg names to all PlumberEndpoint variables in pr$handle() (#677)
rstudio/plumber: NEWS.md

plumber 1.0.0.9999 Development version
--------------------------------------------------------------------------------

### Bug fixes

> * Something about how handle deals with ... in relation to PlumberEndpoint

Reviewer: ```suggestion
* When manually calling a new `Plumber` `handle()`, `envir` may not be supplied when defining a new `PlumberEndpoint` #677  
```

> * When manually calling `Plumber$handle()`, `envir` may not be passed through when defining a new `PlumberEndpoint` #677  

Reviewer: ```suggestion
* When calling `Plumber$handle()` and defining a new `PlumberEndpoint`, `...` will be checked for invalid names #677  
```
